test/simulator/timestamp/src/include/connection_simulator.h

```diff
@@ -68,7 +68,7 @@ class connection_simulator {
     /* Member variables */
     private:
     std::vector<session_simulator *> _session_list;
-    uint64_t _oldest_ts;
-    uint64_t _stable_ts;
-    uint64_t _durable_ts;
+    uint64_t _oldest_ts = 0;
```

Comment:
I think it is better to have an `init` function instead of setting those values in the header. But this is probably a personal pref

Reply:
init for this is private, hence it is never called.

Reply:
Updated in [2be2f8e](https://github.com/wiredtiger/wiredtiger/pull/8398/commits/2be2f8e87575fb24ba9ddea5c127848f308355cf)